agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -270,7 +270,13 @@ class backhaul_manager : public btl::transport_socket_thread {
             beerocks::eFreqType::FREQ_UNKNOWN; /**< Frequency band */
         beerocks::eWiFiBandwidth max_bandwidth =
             beerocks::eWiFiBandwidth::BANDWIDTH_UNKNOWN; /**< Maximum supported bandwidth */
-        beerocks_message::sVapsList vaps_list;           /**< List of VAPs in radio. */
+        uint16_t ht_capability = 0;                      /**< HT capabilities */
+        std::basic_string<uint8_t>
```

Comment:
Why use basic_string and not `std::array`? Isn't it always 16 bytes?

Reply:
Changed to std::array

Reply:
Actually now @arnout introduced a new class `bytes` so perhaps its better to use this one instead (nitpick)